machine_test.go

```diff
@@ -812,17 +812,14 @@ func TestCaptureFifoToFile(t *testing.T) {
 	defer f.Close()
 
 	go func() {
-		t := time.NewTicker(250 * time.Millisecond)
-		select {
-		case <-t.C:
-			f.Close()
-		}
+		t := time.AfterFunc(250*time.Millisecond, func() { f.Close() })
+		<-t.C
```

Comment:
You don't need this.  You can just do `time.AfterFunc()` without the goroutine and without looking at `t.C`.